js/app.js

```diff
@@ -96,55 +92,83 @@ var table = document.createElement('table');
 generateHeader(times, table);
 
 // renders individual stores
-for (i = 0; i < stores.length; i++) {
-  stores[i].render(table, storeMax);
+tbody = document.createElement('tbody');
+for (var i = 0; i < stores.length; i++) {
+  stores[i].render(tbody, storeMax);
 };
+table.appendChild(tbody);
 
 //renders total
 generateFooter(table, totalMax);
-
 article.appendChild(table);
 
 //converts array into table row
 function generateHeader(rowArray, table) {
+  thead = document.createElement('thead');
   var row = document.createElement('tr');
   for (var i = 0; i < times.length; i++) {
     var cell = document.createElement('th');
     cell.textContent = rowArray[i];
-    row.appendChild(cell);
-  }
-  table.appendChild(row);
+    row.appendChild(cell);}
+  thead.appendChild(row);
+  table.appendChild(thead);
 };
 
 //converts array into table row
-function generateFooter(table, maximum) {
+function generateFooter(table) {
+  tfoot = document.createElement('tfoot');
+  tfoot.setAttribute('style', 'color: #990000');
   //generates an array with the totals for any given hour
-  var totals = ['Totals: '];
-  for (i = 1; i < 17; i++) {
-    totals.push(0);
-    for (var j = 0; j < stores.length; j++) {
-      totals[i] += stores[j].savedLog[i];
-    }
-  }
   var row = document.createElement('tr');
   for (var i = 0; i < times.length; i++) {
     var cell = document.createElement('td');
     var span = document.createElement('span');
-    span.textContent = totals[i];
+    span.textContent = storeTotals[i];
     span.setAttribute('style', 'font-weight: bold');
     cell.appendChild(span);
-    if (0 < i && i < 16) {
-      var success = totals[i] / maximum;
-      cell.setAttribute('style', 'background-color: rgba(250, 128, 114, ' + success + ')');
+    if (0 < i && i < 15) {
+      var success = storeTotals[i] / totalMax;
+      cell.setAttribute('style', 'background-color: rgba(255, 100, 80, ' + success + ')');
       var div = document.createElement('div');
       cell.appendChild(div);
       var img = document.createElement('img');
       img.setAttribute('src', 'images/salmon-small.png');
-      img.setAttribute('width', 60 * success);
+      img.setAttribute('width', 60 * Math.pow(success, .75));
       div.appendChild(img);
       cell.appendChild(div);
     }
     row.appendChild(cell);
   }
-  table.appendChild(row);
+  tfoot.appendChild(row);
+  table.appendChild(tfoot);
 };
+
+
+var addStore = document.getElementById('addStore');
+
+
+addStore.addEventListener('submit',
+  function (event) {
+    event.preventDefault();
+    var location = event.target.location.value;
+    var minCust = event.target.minCust.value;
+    var maxCust = event.target.maxCust.value;
+    var aveCookies = event.target.aveCookies.value;
```

Comment:
Your math is not being calculated correctly. You should console.log the typeof of this variable.